screens/forgot-password.tsx

```diff
@@ -197,7 +209,10 @@ export default function TabSubmitScreen() {
               source={require("../assets/images/icon.png")}
```

Comment:
Can you make a follow up to replace this image with an actual bathroom finder icon? I think it would be a nice touch to make the app feel more real. Don't spend a ton of time on it.

Reply:
Issue #168